### PR TITLE
Update UG for FindCommand

### LINKED ISSUE #155: Update UG for FindCommand
- Update Commands for FindCommand 
- Update Commands Summary for FindCommand

### PR DESCRIPTION
**Pull request by jevan13**
## Describe your changes
- Update content of FindCommand to reflect latest changes
- Update image for FindCommand example
- Add tips
- Add common mistakes

## Issue ticket number and link
closes #155 

## Checklist before requesting a review
- [X] I have performed a self-review of my code.
- [X] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -483,20 +483,46 @@
...
   Examples:
-  * `find all John` returns people `john` and `John Doe`
-  * `find e John` returns employees `john` and `John Doe`
-  * `find ph alex david` returns potential hires `Alex Yeoh`, `David Li`<br>
-    ![result for 'find alex david'](images/findAlexDavidResult.png)
+  * `find all n/John p/12345678` returns persons with `John` in their name, and with phone number `12345678`
+  * `find e p/12345678 e/john@example.com alice@example.com` returns employees with phone number `12345678` and with
+  email either `john@example.com` or `alice@example.com`
+  * `find ph d/IT r/SWE Manager` returns potential hires with department `IT`, and role either `SWE` or `Manager`
+![result for 'find alex david'](images/findjohnResult.png)
```
Comment (KiKuasaurus) [inline docs/UserGuide.md]: Should this be [result for `find john`] instead. Should there be another line above it that explains what the image is showing?

Comment (jevan13) [reply docs/UserGuide.md]: Adding a title above! Thanks!

Comment (KiKuasaurus) [reply docs/UserGuide.md]: Should the description for when the image does not load also be changed?

**Review by KiKuasaurus — APPROVED** [review]
LGTM!

**Pull request merged**